public/static/locales/en/common.json

```diff
@@ -98,8 +98,10 @@
     "logout": "Sign out"
   },
   "alert": {
-    "dismissAlert": "Close this alert"
+    "dismissAlert": "Close this alert",
+    "dismissNotification": "Close this alert"
   },
   "continue": "Continue",
-  "error": "Error"
+  "error": "Error",
+  "unknown": "unknown"
```

Comment:
Is capital "U" needed here?

Reply:
Updated to use a capital

https://github.com/cds-snc/platform-forms-client/pull/1467/commits/27e935fbb7d7990ba7353a7b599eb975206bcf67